src/Uno.UI/UI/Xaml/FrameworkElement.Layout.netstd.cs

```diff
@@ -120,6 +120,7 @@ private void InnerMeasureCore(Size availableSize)
 				.Add(marginSize)
 				// Making sure after adding margins that clipped DesiredSize is not bigger than the AvailableSize
 				.AtMost(availableSize)
+				.AtMost(maxSize)
```

Comment:
could also just use `AtMost(frameworkAvailableSize)` instead, as that one is already clamped under `maxSize`

Reply:
But actually the `frameworkAvailableSize` has margin subtracted, so I am guessing that would not work properly as here we re-add it back 🤔 ?